services/src/main/java/org/keycloak/authentication/authenticators/browser/PasskeysConditionalUIAuthenticator.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/**
+ * @author <a href="mailto:takashi.norimatsu.ws@hitachi.com">Takashi Norimatsu</a>
+ */
+package org.keycloak.authentication.authenticators.browser;
+
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.models.KeycloakSession;
+
+import jakarta.ws.rs.core.Response;
+
+public class PasskeysConditionalUIAuthenticator extends WebAuthnPasswordlessAuthenticator {
+
+    public PasskeysConditionalUIAuthenticator(KeycloakSession session) {
+        super(session);
+    }
+
+    @Override
+    public void authenticate(AuthenticationFlowContext context) {
+        super.authenticate(context);
+        Response challenge = context.form()
+                .createForm("passkeys-condional-authenticate.ftl");
```

Comment:
I would expect that the form would be called `login-passkeys-condional-authenticate.ftl` but then I 
just noticed that not all authenticator / login related ftl templates are prefixed with `login-`. 

So it seems that this inconsitency is not new and thus not a problem. Changing this now (for the other forms) would probably break existing customizations. Therefore it is okay to keep the template name.

https://github.com/keycloak/keycloak/tree/main/themes/src/main/resources/theme/base/login

Reply:
Thank you your advice. I will rename the template file as you recommended.